phpcs.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Beans">
+    <description>The code standard for Beans.</description>
+    <file>.</file>
+    <exclude-pattern>dist/*</exclude-pattern>
```

Comment:
Not sure if you plan to distribute this project as zip file, but if so, the build can be done to the de facto directory of `dist/`.